### PR TITLE
added nestjs client library

### PR DESCRIPTION
**Pull request by danduh**
We just the added ability to generate a client library dedicated to NestJS projects.

### CODE REVIEW

File: lib/TypescriptClientGenerator.php [hunk 1]
```diff
@@ -11,7 +11,7 @@ class TypescriptClientGenerator extends ClientGeneratorFromXml
     private $_framework;
     private $_targetKalturaServer;
 
-	function __construct($xmlPath, Zend_Config $config, $framework = "typescript", $srcAPIBasePath = "src/api")
+	function __construct($xmlPath, Zend_Config $config, $framework = "nestjs", $srcAPIBasePath = "src/api")
```
[the remark this answers is not in the document]
Comment (esakal) [reply lib/TypescriptClientGenerator.php]: @danduh make sure you address this comment

[the remark this answers is not in the document]
Comment (danduh) [reply lib/TypescriptClientGenerator.php]: Fixed

**Comment by esakal** [comment]
> I suggest you to build angular & typescript libraries prior your change and then with your change and do a folder comparison just to make sure it output the same code for the existing ngx and typescript libraries.

did you compare the output of the new/old generators.

anyhow it seems fine, once you compare it I approve :)

@mosheMaorKaltura?

**Comment by danduh** [comment]
Yes, the output of `/types` is identical

**Pull request merged**